docs/integreatly/modules/ROOT/pages/ds-connect-client-to-server.adoc

```diff
@@ -0,0 +1,71 @@
+= Connecting the {sync-service} client to your {sync-service} server application
+:doctype: book
+
+.Prerequisites
+
+* You have followed the documentation to deploy your {sync-service} server application.
+* You have set up a web project that supports ES6. For example:
+** Using https://reactjs.org/docs/create-a-new-react-app.html[Create React App]
+** Using https://ionicframework.com/getting-started[Ionic Getting Started]
+** Using https://angular.io/start[Getting Started with Angular]
+** Using https://webpack.js.org/guides/getting-started/[Webpack Getting Started Guide]
+
+= Procedure
+
+. Get the hostname of the {sync-service} Server application.
+.. Run the command
++
+[source,bash]
+----
+oc get route <data-sync-application-name>
+----
++
+.. Check the output:
++
+[source,bash]
+----
+NAME                             HOST/PORT               PATH      SERVICES        PORT      TERMINATION   WILDCARD
+<sync-server-application-name>   <sync-server-hostname>            data-sync-app   <all>                   None
+----
++
+.. Record the value for `<sync-server-hostname>`.
+. Add the following libraries to your project:
++
+[source,javascript]
+----
+npm install @aerogear/voyager-client
+npm install graphql
+npm install graphql-tag
+----
++
+. In your project source code, import and configure the client using the server hostname.
```

Comment:
wondering: isn't it more typical that user would be modifying their existing local server, than doing this for first time?

Reply:
Do you mean modifying their existing client app? That is a fair point. I can change it to something like `if you haven't done so already, import and configure the client using the server hostname`